platooning_strategic_IHP/src/platoon_manager_ihp.cpp

```diff
@@ -709,7 +776,7 @@ namespace platoon_strategic_ihp
         */
 
         double min_diff = 99999.000;
-        int cut_in_index = -1;
+        int cut_in_index = -2;
```

Comment:
This should remain -1.  The value -2 doesn't belong anywhere.  Even though the below loop is broken (a different problem), it will return this value if joiner is in front of everyone.

Reply:
Agreed. Set -1 as default value is the correct value.